docs/integrations/sources/bing-ads.md

```diff
@@ -2,22 +2,44 @@
 
 This page contains the setup guide and reference information for the Bing Ads source connector.
 
+## Prerequisites
+- Microsoft Advertising account
+- Microsoft Developer Token
+- Reports start date
+
 ## Setup guide
 
-### Step 1: Set up Bing Ads
+<!-- env:oss -->
+
+For Airbyte Open Source set up your application to get **Client ID**, **Client Secret**, **Refresh Token** 
 
 1. [Register your application](https://docs.microsoft.com/en-us/advertising/guides/authentication-oauth-register?view=bingads-13) in the Azure portal.
 2. [Request user consent](https://docs.microsoft.com/en-us/advertising/guides/authentication-oauth-consent?view=bingads-13l) to get the authorization code.
 3. Use the authorization code to [get a refresh token](https://docs.microsoft.com/en-us/advertising/guides/authentication-oauth-get-tokens?view=bingads-13).
 
-:::note
+4. :::note
```

Comment:
```suggestion
:::note
```
The note block is not rendered correctly otherwis
<img width="1071" alt="Screenshot 2023-11-03 at 18 39 52" src="https://github.com/airbytehq/airbyte/assets/5551758/da8c3f1c-aa46-4b06-a0a5-7cb780efeb75">
e

Reply:
fixed